automol/util/vector.py

```diff
@@ -69,40 +68,39 @@ def are_parallel(xyz1, xyz2, orig_xyz=(0.0, 0.0, 0.0), anti=False, tol=1e-3):
 
 
 def orthogonalize(xyz1, xyz2, normalize=False):
-    """orthogonalize `xyz2` against `xyz1`"""
-    overlap = numpy.dot(xyz1, xyz2)
-    norm = numpy.dot(xyz1, xyz1)
-    oxyz2 = numpy.subtract(xyz2, numpy.multiply(overlap / norm, xyz1))
+    """Orthogonalize `xyz2` against `xyz1`.
+    :param xyz1: 3D vector
+    :param xyz2: 3D vector.
+    """
+    overlap = np.dot(xyz1, xyz2)
+    norm = np.dot(xyz1, xyz1)
+    oxyz2 = np.subtract(xyz2, np.multiply(overlap / norm, xyz1))
     if normalize:
         oxyz2 = unit_norm(oxyz2)
     return oxyz2
 
 
-def flip_if_left_handed(xvec, yvec, zvec) -> Vector:
+def flip_if_left_handed(xvec: Vector, yvec: Vector, zvec: Vector) -> Vector:
     """Given three vectors, flips the third one, if necessary, to make a right-handed
-    coordinate system
+    coordinate system.
 
     The vectors need not be orthogonal, but they must be linearly independent for this
     to work.  Does not change direction -- only flips the sign, if necessary.
 
     :param xvec: A vector defining the x direction
-    :type xvec: Vector
     :param yvec: A vector defining the y direction
-    :type yvec: Vector
     :param zvec: A vector defining the z direction
-    :type zvec: Vector
     :returns: `zvec` or, if they made a left-handed system, `-zvec`
-    :rtype: Vector
     """
-    zdir = numpy.cross(xvec, yvec)
-    proj = numpy.dot(zvec, zdir)
+    zdir = np.cross(xvec, yvec)
+    proj = np.dot(zvec, zdir)
     if proj < 0.0:
-        zvec = numpy.negative(zvec)
+        zvec = np.negative(zvec)
     return tuple(map(float, zvec))
```

Comment:
Based on the new definition of `Vector` as a simple alias for `np.ndarray`, we should simply return the `zvec` as-is:
```
return zvec
```